### DISCUSSION TITLE
Question about Criminal Pacer Districts

### DISCUSSION THREAD

**Discussion by dksentencingstats** [freelawproject/courtlistener#4685] [Q&A]
We've noticed that in some districts, for instance Tennessee Eastern, Juriscraper is returning an error on every query. Manually testing the docket IDs reveals that the district is returning an array of sub-docket IDs, even though there is really only one docket ID and one case name. It is displaying each defendant as it's own row, within the primary cases. 

For example: https://ecf.tned.uscourts.gov/cgi-bin/DktRpt.pl?docket_number=3:19-CR-00056

When you click "find case" on this URL, it is returning the list below.  

Does Juriscraper have the logic to know to select just the first row as the case name? Is there some change we should make so it recognizes this? Also is it equipped to scrape each of the listed defendants?

Select a case:
3:19-cr-00056-TAV-JEM USA v. Harris, et al.,-TV2 (closed 02/22/2022) -
3:19-cr-00056-TAV-HBG-1 Darrle Wade Harris (closed 12/20/2021)
3:19-cr-00056-TAV-HBG-2 Cordarryl Wade Harris (closed 06/17/2020)
3:19-cr-00056-TAV-JEM-3 Ronald Anthony Thompson (closed 02/14/2020)
3:19-cr-00056-TAV-HBG-4 Deairrus Jamar Taylor (closed 07/16/2020)
3:19-cr-00056-TAV-HBG-5 DeSean Lee Greene (closed 05/07/2021)
3:19-cr-00056-TAV-HBG-6 Jamal Mylen McDonald (closed 02/28/2020)
3:19-cr-00056-TAV-HBG-7 Raynell Eugene McDonald (closed 06/25/2021)
3:19-cr-00056-TAV-JEM-8 Raphiel Antwon Thompson (closed 02/22/2022)
3:19-cr-00056-TAV-HBG-9 Robert Lee-Rogener McDonald (closed 11/23/2020)
3:19-cr-00056-TAV-HBG-10 Keith Madison (closed 03/10/2021)
3:19-cr-00056-TAV-JEM-11 Sherard Ramon Taylor (closed 03/25/2021)

**Comment by mlissner** [comment]
There's some logic in JS to handle some of the stuff you're describing, but can you share your code or which parts of JS you're using? You say it is returning an error, but can you share what that error is, please?
